pkg/scheduler/framework/interface_test.go

```diff
@@ -93,30 +99,37 @@ func assertStatusCode(t *testing.T, code Code, value int) {
 
 func TestPluginToStatusMerge(t *testing.T) {
 	tests := []struct {
+		name      string
 		statusMap PluginToStatus
 		wantCode  Code
 	}{
 		{
+			name:      "merge Error and Unschedulable statuses",
 			statusMap: PluginToStatus{"p1": NewStatus(Error), "p2": NewStatus(Unschedulable)},
 			wantCode:  Error,
 		},
 		{
+			name:      "merge Success and Unschedulable statuses",
 			statusMap: PluginToStatus{"p1": NewStatus(Success), "p2": NewStatus(Unschedulable)},
 			wantCode:  Unschedulable,
 		},
 		{
+			name:      "merge Success, UnschedulableAndUnresolvable and Unschedulable statuses",
 			statusMap: PluginToStatus{"p1": NewStatus(Success), "p2": NewStatus(UnschedulableAndUnresolvable), "p3": NewStatus(Unschedulable)},
 			wantCode:  UnschedulableAndUnresolvable,
 		},
 		{
+			name:     "merge nil status",
 			wantCode: Success,
 		},
 	}
 
 	for i, test := range tests {
```

Comment:
Let's remove the `i` and `test $i` below (L131)

Reply:
All unnecessary test scenario information in error has been removed.